simcase-game/src/components/ScenarioForm.js

```diff
@@ -4,6 +4,7 @@ import { jsx } from "@emotion/core";
 
 import ImageContainer from "../components/ImageContainer";
 import TaskList from "../components/TaskList";
+import ProgreesBar from "../components/ProgressBar";
```

Comment:
Here you got a typo, because it should be "ProgressBar" and you wrote "ProgreesBar"